playbooks/common/openshift-cluster/openshift_hosted.yml

```diff
@@ -27,21 +27,21 @@
       logging_elasticsearch_ops_cluster_size: "{{ openshift_hosted_logging_elasticsearch_ops_cluster_size | default(1) }}"
   roles:
   - role: openshift_hosted
-  - role: openshift_hosted_metrics
+  - role: openshift_metrics
```

Comment:
Actually, I did intend to modify this entry because of the logging issue; I assume there would be the same problem with metrics.  The user reported an issue installing logging on 3.5 using these playbooks, which point to the wrong playbooks.